_posts/en/posts/2018-12-25-release-0.17.1.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Bitcoin Core 0.17.1 Released
+name: blog-release-0.17.1
+id: en-blog-release-0.17.1
+lang: en
+type: posts
+layout: post
+share: true
+
+## If this is a new post, reset this counter to 1.
+version: 1
+
+excerpt: >
+  Bitcoin Core 0.17.1, a maintenance release bringing bug fixes and minor
+  improvements, is now available.
+---
+Bitcoin Core version 0.17.1 is now available for [download][download
+page] containing several bug fixes and minor improvements.  For a
+complete list of changes, please see the [release notes][].
+
+If have any questions, please stop by our [IRC chatroom][irc] and we’ll
```

Comment:
I know this is the same wording as previous release announcements. I think future announcements could be improved by specifying which IRC channel users should go to (presumably #bitcoin since these will be end-user questions).